### PR TITLE
docs(GraphQL): Add paragraph around combining ID with @id

### PR DESCRIPTION
**Pull request by verneleem**
- Add information on relationship between ID and @id
- Add example of get query when using `@id` directive
- Typo fixes
- Rephrased a sentence that was using `id` in the @id directive paragraph.
@aaroncarey patched commits 2-5 to edit the full page for TW style, improve the introduction, and move a forward-looking statement to the comments.

Fixes DOC-115

### CODE REVIEW

File: content/graphql/schema/ids.md [hunk 1]
```diff
@@ -50,7 +50,27 @@ Identities created with `@id` are reusable - if you delete an existing user, you
 
 Fields with the `@id` directive must have the type `String!`.
 
-As with `ID` types, Dgraph will generate queries and mutations so you'll also be able to query, update and delete by id.
+As with `ID` types, Dgraph will generate queries and mutations so you'll also be able to query, update and delete by the field with the @id directive.
```
Comment (aaroncarey) [inline content/graphql/schema/ids.md:53]: Dgraph will generate -> Dgraph generates
you'll also be able to query -> you can query
by the field with the @id directive. -> data in fields specified with the @id directive.

**Review by aaroncarey — COMMENTED** [review]
Looks good; I'm patching-in to this PR to make a few changes (so no need to act on my comments here).

**Review by aaroncarey — APPROVED** [review]
Ready to merge, unless you see issues with my edits.

**Comment by verneleem** [comment]
LGTM. @aaroncarey, I don't have access to merge still. Don't know if that was dorected at me or the other reviewers.

**Comment by aaroncarey** [comment]
> LGTM. @aaroncarey, I don't have access to merge still. Don't know if that was directed at me or the other reviewers.

Thanks for the LGTM, I just wanted a second set of eyes on my edits before merging. Thanks for your contribution!

**Pull request merged**